client/src/app/tabs/MultiSheetTab.js

```diff
@@ -315,31 +315,16 @@ export class MultiSheetTab extends CachedComponent {
   }
 
   componentDidMount() {
-    const {
```

Comment:
Seems like we are going back and forth here.

Could you verify that the bug fix that introduced this change (if one exists, check `git blame`) is still fixed when removing that line?